blog/index.html

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I think you also need to add code to [/_includes/sidebar.html)](https://github.com/datreant/datreant.github.io/blob/master/_includes/sidebar.html) for the blog to show up. I just looked at the MDA pages and it has stuff like
```
      {% assign blogpagedir = site.paginate_path | split: ":" | first %}
      {% assign blogpagedir_len = blogpagedir | size %}
      {% assign pages_list = site.pages %}
      {% for node in pages_list %}
        {% if node.title != null %}
          {% if node.layout == "page" %}
            {% assign prefix = node.url | truncate: blogpagedir_len, '' %}
            {% if prefix != blogpagedir %}
              <a class="sidebar-nav-item{% if page.url == node.url %} active{% endif %}" href="{{ node.url }}">{{ node.title }}</a>
            {% endif %}
          {% endif %}
        {% endif %}
      {% endfor %}
```

Reply:
It shows up...I believe I added it? Don't recall...